Dockerfile

```diff
@@ -1,28 +1,46 @@
-FROM gradle:jdk-alpine
+# FROM gradle:jdk-alpine
 
-ARG SBT_VERSION=1.2.8
+### Taken from: https://github.com/mozilla/docker-sbt
+
+ARG OPENJDK_TAG=8u232
+FROM openjdk:${OPENJDK_TAG}
+
+ARG SBT_VERSION=1.4.5
+
+# Install sbt
+RUN \
+  mkdir /working/ && \
+  cd /working/ && \
+  curl -L -o sbt-$SBT_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb && \
+  apt-get update && \
+  apt-get install sbt && \
+  cd && \
+  rm -r /working/ && \
+  sbt sbtVersion
 
 USER root
 
 # Install sbt
-RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing sbt
+# RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing sbt
 
-COPY --chown=gradle:gradle . .
+# COPY --chown=gradle:gradle . .
 #RUN sbt -Dsbt.ivy.home=.ivy2 -Dsbt.global.base=.sbt/1.0 -Dsbt.boot.directory=.sbt/boot test
 RUN mkdir -p /opt/scala-test-runner
 RUN mkdir -p /opt/scala-test-runner/lib
 
 #RUN cp /home/gradle/target/scala-2.12/*.jar /opt/scala-test-runner/lib
 COPY bin/run.sh /opt/scala-test-runner/bin/
+# RUN sbt compile
```

Comment:
This will need to be uncommented at some point at this compiles the actual test runner.